src/buildstream/_frontend/cli.py

```diff
@@ -552,7 +555,29 @@ def build(
             _PipelineSelection.ALL,
         ],
     ),
-    help="The dependencies to show",
+    help="Types of Elements to Show",
+)
+@click.option(
+    "--kind",
+    "-k",
+    default=[],
+    show_default=True,
+    multiple=True,
+    type=FastEnumType(
+        _ElementKind,
```

Comment:
"Element Kind" is definitely not how you should name this. The element kind is already a concept in buildstream, and it's not this.

I don't know what would be the best way to name this option though. info? keys?

Let's wait for other maintainer's opinions, they might have a better idea.